images/mnt.proto

```diff
@@ -32,29 +32,34 @@ enum fstype {
 	CGROUP2			= 23;
 };
 
+message vendor_mnt_entry {
+};
+
 message mnt_entry {
-	required uint32		fstype			= 1;
-	required uint32		mnt_id			= 2;
-	required uint32		root_dev		= 3 [(criu).dev = true];
-	required uint32		parent_mnt_id		= 4;
-	required uint32		flags			= 5 [(criu).hex = true];
+	required uint32			fstype			= 1;
+	required uint32			mnt_id			= 2;
+	required uint32			root_dev		= 3 [(criu).dev = true];
+	required uint32			parent_mnt_id		= 4;
+	required uint32			flags			= 5 [(criu).hex = true];
 
-	required string		root			= 6;
-	required string		mountpoint		= 7;
-	required string		source			= 8;
-	required string		options			= 9;
+	required string			root			= 6;
+	required string			mountpoint		= 7;
+	required string			source			= 8;
+	required string			options			= 9;
 
-	optional uint32		shared_id		= 10;
-	optional uint32		master_id		= 11;
+	optional uint32			shared_id		= 10;
+	optional uint32			master_id		= 11;
 
-	optional bool		with_plugin		= 12;
-	optional bool		ext_mount		= 13;
+	optional bool			with_plugin		= 12;
+	optional bool			ext_mount		= 13;
 
-	optional string		fsname			= 14;
-	optional bool		internal_sharing	= 15;
+	optional string			fsname			= 14;
+	optional bool			internal_sharing	= 15;
 
-	optional bool		deleted			= 16;
-	optional uint32		sb_flags		= 17 [(criu).hex = true];
+	optional bool			deleted			= 16;
+	optional uint32			sb_flags		= 17 [(criu).hex = true];
 	/* user defined mapping for external mount */
-	optional string		ext_key			= 18;
+	optional string			ext_key			= 18;
+
+	optional vendor_mnt_entry	vendor			= 1024;
```

Comment:
I won't insist, but I'd prefer an unaligned `vendor_mnt_entry` rather than all this indentation churn.

Reply:
Indentation churn may go in separate patch. And, while we're all here, 4 of these patches can be merged together -- no need to add vendor field to different images with different patches.

Reply:
@rppt
I can get rid of proper indentation, not a problem.

Reply:
@xemul I will merge 3 (not 4, please see my reply in images/sk-unix.proto) patches.